proknow-sdk/Requestor.cs

```diff
@@ -300,6 +301,8 @@ internal async Task<string> GetDomainStatusAsync()
             string hasMore = "true";
             string next = null;
             var patients = new List<string>();
+            var match = Regex.Match(route, @"^/workspaces/[a-fA-F0-9]{32}/patients$");
+            var isPatientsRoute = match.Success && method == HttpMethod.Get;
```

Comment:
Instead of a regex match, what do you think about making the determination about what kind of paging strategy to use with an additional parameter? Like `strategy` with possible string values of `"numbered"` and `"next"` (default value of `"next"` perhaps)?